scraper/process-data.js

```diff
@@ -3,32 +3,16 @@
 // Reads and parses anchor data files from bikeshed-data repository
 // and writes data.json containing parsed and formatted data
 
-const { readFile, readdirSync, writeFile } = require("fs");
+const { readdirSync } = require("fs");
```

Comment:
Why no async this one too?

Reply:
ah. VS code wasn't showing suggestion, thought it didn't exist. will fix.